packages/frontend/src/build/createApi.ts

```diff
@@ -77,13 +77,27 @@ function getActivityChart(
   apiChart: ActivityApiChart,
   ethereumChart: ActivityApiChart,
 ): FrontendActivityChart {
-  const length = Math.min(apiChart.data.length, ethereumChart.data.length)
+  const lastProjectTimestamp = apiChart.data.at(-1)?.[0]
+  if (!lastProjectTimestamp) {
+    throw new Error('No data in activity chart')
+  }
+  const ethChartTimestampIndex = ethereumChart.data.findIndex(
+    (x) => x[0].toNumber() === lastProjectTimestamp.toNumber(),
+  )
+  if (ethChartTimestampIndex === -1) {
+    throw new Error('No matching timestamp in ethereum chart')
+  }
+  const alignedEthChartData = ethereumChart.data.slice(
+    0,
+    ethChartTimestampIndex + 1,
+  )
+  const length = Math.min(apiChart.data.length, alignedEthChartData.length)
```

Comment:
are we sure that both Ethereum and projects data start from the same timestamp?

Reply:
Fair point, tested, works :)